src/Drivers/LaravelHttpServer.php

```diff
@@ -222,22 +222,35 @@ private function handleRequest(AmpRequest $request): Response
             Execution::instance()->tick();
         }
 
-        $filepath = public_path($path = $request->getUri()->getPath());
-
-        if (file_exists($filepath) && ! is_dir($filepath)) {
+        $uri = $request->getUri();
+        $path = $uri->getPath() ?: '/';
+        $query = $uri->getQuery() ?? '';
+        $fullPath = $path . ($query !== '' ? '?' . $query : '');
+        $absoluteUrl = rtrim($this->url(), '/') . $fullPath;
+
+        $filepath = public_path($path);
+        if (file_exists($filepath) && !is_dir($filepath)) {
             return $this->asset($filepath);
         }
 
         $kernel = app()->make(HttpKernel::class);
 
+        $contentType = $request->getHeader('content-type')[0] ?? '';
+        $method = strtoupper($request->getMethod());
+        $rawBody = (string)$request->getBody();
+        $parameters = [];
+        if ($method !== 'GET' && str_starts_with(strtolower($contentType), 'application/x-www-form-urlencoded')) {
+            parse_str($rawBody, $parameters);
+        }
+
         $symfonyRequest = Request::create(
-            $path,
-            $request->getMethod(),
-            $request->getQueryParameters(),
+            $absoluteUrl,
+            $method,
+            [$parameters],
```

Comment:
No need to wrap `$parameters` in array here as the variable is an array already.
```suggestion
            $parameters,
```